app/src/main/java/org/openobservatory/ooniprobe/utils/NotificationService.java

```diff
@@ -80,13 +80,16 @@ public void sendRegistrationToServer() {
         final String auth_secret_file = context.getFilesDir() + "/orchestration_secret.json";
 
         //LOGGING
+        /*
         System.out.println("probe_cc: " + geoip_country_path);
         System.out.println("probe_asn: " + geoip_asn_path);
         System.out.println("platform: " + platform);
         System.out.println("software_name: " + software_name);
         System.out.println("software_version: " + software_version);
         System.out.println("supported_tests: " + supported_tests);
         System.out.println("token: " + device_token);
+        */
+        if (device_token == null) return;
```

Comment:
Maybe we can log the fact that the device is `null` so we know when we're debugging?

Reply:
This scenario should happen only if user deacivated the notification.

Reply:
Okay, maybe a comment should then be useful to clarify _when_ this could happen